src/main/webapp/style.css

```diff
@@ -93,9 +93,78 @@ input[type=submit]:hover {
   display: none;
 }
 
-#buttons {
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  border-radius: 34px;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
```

Comment:
This is so nitpicky so feel free to disregard but right now the div's visibility changes instantaneously even though toggling the map takes .4 seconds. It would be cool to create some fade in animations that correspond to the toggle for each div, but this is obviously pretty low priority

Reply:
Yeah a fade in would look nice, will look into doing this